### ISSUE TITLE
rapidjson/reader.h - range check bug?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/rapidjson#6]
```
I confirmed that it is a bug. Thank you for reporting.
```

Original issue reported on code.google.com by `xushiweizh` on 6 Dec 2011 at 2:11

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r51.
```

Original comment by `milo...@gmail.com` on 6 Dec 2011 at 4:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision bedeb0d8ea55.
```

Original comment by `milo...@gmail.com` on 6 Jun 2014 at 6:04
